modules/veteran/app/sidekiq/rep_addresses/xlsx_file_processor.rb

```diff
@@ -0,0 +1,162 @@
+# frozen_string_literal: true
+
+module RepAddresses
+  class XlsxFileProcessor
+    include SentryLogging
+
+    SHEETS_TO_PROCESS = %w[Attorneys Representatives].freeze
```

Comment:
Based on Martin's most recent email, we probably want to keep agents in here

Reply:
I'll add agents in a separate PR if we need to.